pegasus/sites.v3/code.org/public/thebadguys.md.partial

```diff
@@ -15,18 +15,18 @@ social:
 
 # Code with The Bad Guys
 
-{{ tutorial_promo2, heading: "", background_image: "/images/marketing/landingpage-thebadguys-banner.png", text_color: "white", box_background_color: "rgb(0,0,0,0.4)", left_heading: "Code with <em>The Bad Guys</em>", left_description: "Make a story or animation starring the crew from <strong>The Bad Guys</strong> movie!", left_specs: "", left_button_url: "https://studio.code.org/projects/thebadguys/new", left_button_text: "Start Now", right_heading: "New to coding?", right_description: "Try a tutorial, then come back to make a project with <strong>The Bad Guys</strong>.", right_specs: "", right_button_url: "https://studio.code.org/s/hello-world-food-2021/reset", right_button_text: "Learn First" }}
+{{ tutorial_promo2, heading: "", background_image: "/images/marketing/landingpage-thebadguys-banner.png", alt_text: "Background image featuring a car driven by animated characters from the film The Bad Guys", text_color: "white", box_background_color: "rgb(0,0,0,0.4)", left_heading: "Code with <em>The Bad Guys</em>", left_description: "Make a story or animation starring the crew from <strong>The Bad Guys</strong> movie!", left_specs: "", left_button_url: "https://studio.code.org/projects/thebadguys/new", left_button_text: "Start Now", right_heading: "New to coding?", right_description: "Try a tutorial, then come back to make a project with <strong>The Bad Guys</strong>.", right_specs: "", right_button_url: "https://studio.code.org/s/hello-world-food-2021/reset", right_button_text: "Learn First" }}
```

Comment:
This `alt_text` attribute already existed on the `tutorial_promo2` view so I added a description here too.

Reply:
I love this alt text! Do you think it needs to be translated?

Reply:
These entire files get translated (this came up during HoC so it's fresh in my mind haha!)

Reply:
Ah, cool beans!